diff --git a/config.def.h b/config.def.h
index 6ebea98..2b7c693 100644
--- a/config.def.h
+++ b/config.def.h
@@ -6,7 +6,24 @@
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
 static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
-static int borderpx = 2;
+
+/* Spare fonts */
+
+static char *font2[] = {
+	"Inconsolata for Powerline:pixelsize=12:antialias=true:autohint=true",
+	"Hack Nerd Font Mono:pixelsize=11:antialias=true:autohint=true",
+};
+
+
+/* disable bold, italic and roman fonts globally */
+
+int disablebold = 1;
+
+int disableitalic = 1;
+
+int disableroman = 1;
+
+static int borderpx = 10;
 
 /*
  * What program is execed by st depends of these precedence rules:
@@ -56,6 +73,22 @@ static unsigned int blinktimeout = 800;
  */
 static unsigned int cursorthickness = 2;
 
+/*
+ * 1: render most of the lines/blocks characters without using the font for
+ *    perfect alignment between cells (U2500 - U259F except dashes/diagonals).
+ *    Bold affects lines thickness if boxdraw_bold is not 0. Italic is ignored.
+ * 0: disable (render all U25XX glyphs normally from the font).
+ */
+
+const int boxdraw = 1;
+
+const int boxdraw_bold = 1;
+
+
+/* braille (U28XX):  1: render as adjacent "pixels",  0: use font */
+
+const int boxdraw_braille = 1;
+
 /*
  * bell volume. It must be a value between -100 and 100. Use 0 for disabling
  * it
