diff --git a/config.def.h b/config.def.h
index 1c0b587..48b5e7e 100644
--- a/config.def.h
+++ b/config.def.h
@@ -1,21 +1,40 @@
 /* See LICENSE file for copyright and license details. */
 
 /* appearance */
-static const unsigned int borderpx  = 1;        /* border pixel of windows */
+static const unsigned int borderpx  = 2;        /* border pixel of windows */
+static const unsigned int gappx     = 10;       /* gaps between windows */
 static const unsigned int snap      = 32;       /* snap pixel */
 static const int showbar            = 1;        /* 0 means no bar */
 static const int topbar             = 1;        /* 0 means bottom bar */
-static const char *fonts[]          = { "monospace:size=10" };
-static const char dmenufont[]       = "monospace:size=10";
+static const int horizpadbar          = 0;   /* horizontal padding for statusbar */
+static const int vertpadbar           = 8;   /* vertical padding for statusbar */
 static const char col_gray1[]       = "#222222";
 static const char col_gray2[]       = "#444444";
 static const char col_gray3[]       = "#bbbbbb";
 static const char col_gray4[]       = "#eeeeee";
 static const char col_cyan[]        = "#005577";
+static const char *fonts[]            = {
+    "crisp:size=12:antialias=true:autohint=true",
+    "siji:size=12:antialias=true:autohint=true"
+};
+static const char dmenufont[]         = "crisp:size=12:antialias=true:autohint=true";
+static const char normfgcolor[]       = "#f8f6f2";
+static const char normbgcolor[]       = "#1c1b1a";
+static const char normbordercolor[]   = "#1c1b1a";
+static const char selfgcolor[]        = "#0a9dff";
+static const char selbgcolor[]        = "#1c1b1a";
+static const char selbordercolor[]    = "#0a9dff";
+static const char titlefgcolor[]      = "#f8f6f2";
+static const char titlebgcolor[]      = "#1c1b1a";
+static const char titlebordercolor[]  = "#1c1b1a";
+static const char col_black[]         = "#141413";
+static const char col_red[]           = "#ff2c4b";
+static const char col_yellow[]        = "#fade3e";
+static const char col_white[]         = "#d9cec3";
 static const char *colors[][3]      = {
 	/*               fg         bg         border   */
-	[SchemeNorm] = { col_gray3, col_gray1, col_gray2 },
-	[SchemeSel]  = { col_gray4, col_cyan,  col_cyan  },
+	[SchemeNorm]  = { col_gray3, col_gray1, col_gray2 },
+	[SchemeSel]   = { col_gray4, col_cyan, col_cyan },
 };
 
 /* tagging */
@@ -29,12 +48,13 @@ static const Rule rules[] = {
 	/* class      instance    title       tags mask     isfloating   monitor */
 	{ "Gimp",     NULL,       NULL,       0,            1,           -1 },
 	{ "Firefox",  NULL,       NULL,       1 << 8,       0,           -1 },
+	{ "shortcuts",  NULL,      "shortcuts",       0,     1,           -1 },
 };
 
 /* layout(s) */
-static const float mfact     = 0.55; /* factor of master area size [0.05..0.95] */
+static const float mfact     = 0.50; /* factor of master area size [0.05..0.95] */
 static const int nmaster     = 1;    /* number of clients in master area */
-static const int resizehints = 1;    /* 1 means respect size hints in tiled resizals */
+static const int resizehints = 0;    /* 1 means respect size hints in tiled resizals */
 
 static const Layout layouts[] = {
 	/* symbol     arrange function */
@@ -44,7 +64,7 @@ static const Layout layouts[] = {
 };
 
 /* key definitions */
-#define MODKEY Mod1Mask
+#define MODKEY Mod4Mask
 #define TAGKEYS(KEY,TAG) \
 	{ MODKEY,                       KEY,      view,           {.ui = 1 << TAG} }, \
 	{ MODKEY|ControlMask,           KEY,      toggleview,     {.ui = 1 << TAG} }, \
@@ -56,13 +76,23 @@ static const Layout layouts[] = {
 
 /* commands */
 static char dmenumon[2] = "0"; /* component of dmenucmd, manipulated in spawn() */
-static const char *dmenucmd[] = { "dmenu_run", "-m", dmenumon, "-fn", dmenufont, "-nb", col_gray1, "-nf", col_gray3, "-sb", col_cyan, "-sf", col_gray4, NULL };
+static const char *dmenucmd[] = { "dmenu_run", "-l", "15", "-m", dmenumon, "-fn", dmenufont, "-nb", col_gray1, "-nf", col_gray3, "-sb", col_cyan, "-sf", col_gray4, NULL };
 static const char *termcmd[]  = { "st", NULL };
+static const char *passmenucmd[]  = { "passmenu", "-i", "-l", "15", NULL };
+static const char *rangercmd[]  = { "st", "-e", "ranger", NULL };
+static const char *notescmd[]  = { "st", "-e", "vim", "/home/owen/Nextcloud/notes/", NULL };
+static const char *shortcutscmd[]  = { "st", "-c", "shortcuts", "-T", "shortcuts", "-e", "less", "/home/owen/Nextcloud/notes/shortcuts.markdown", NULL };
+static const char *bookmarkscmd[]  = { "/home/owen/source/dotfiles/scripts/dmenu-bm3.sh", NULL };
 
 static Key keys[] = {
 	/* modifier                     key        function        argument */
 	{ MODKEY,                       XK_p,      spawn,          {.v = dmenucmd } },
+	{ MODKEY|ShiftMask,             XK_p,      spawn,          {.v = passmenucmd } },
 	{ MODKEY|ShiftMask,             XK_Return, spawn,          {.v = termcmd } },
+	{ MODKEY|ShiftMask,             XK_slash,  spawn,          {.v = shortcutscmd } },
+	{ MODKEY,                       XK_e,      spawn,          {.v = rangercmd } },
+	{ MODKEY,                       XK_n,      spawn,          {.v = notescmd } },
+	{ MODKEY|ShiftMask,             XK_b,      spawn,          {.v = bookmarkscmd } },
 	{ MODKEY,                       XK_b,      togglebar,      {0} },
 	{ MODKEY,                       XK_j,      focusstack,     {.i = +1 } },
 	{ MODKEY,                       XK_k,      focusstack,     {.i = -1 } },
